test/unit/number-formatting/format-spec.js

```diff
@@ -40,6 +40,11 @@ describe('NumberFormatting format', () => {
             expect(formatNumberNoRounding(-1.126, 1, 2)).toEqual('-1.13');
             expect(formatNumberNoRounding(-1.1, 4, 5)).toEqual('-1.1000');
             expect(formatNumberNoRounding(-5e-7, 4, 8)).toEqual('-0.0000005');
+
+            expect(formatNumberNoRounding(-1.1212, 1, 0)).toEqual('-1');
+            expect(formatNumberNoRounding(-1.126, 1, 0)).toEqual('-1');
+            expect(formatNumberNoRounding(-1.1, 4, 0)).toEqual('-1');
+            expect(formatNumberNoRounding(-5e-7, 4, 0)).toEqual('0');

```

Comment:
is `formatNumberNoRounding(1.12345678, null, 4)` a valid use case? as in "I don't care what's the minimum amount of decimals as long as they aren't more than 4". If it is, right now it would equal `1` due to the parameter default, right?

Reply:
Nevermind, I'm dumb 🤦‍♂️  a couple unit tests would be nice though :)